sechub-pds-solutions/iac/helm/pds-iac/templates/deployment.yaml

```
@@ -0,0 +1,124 @@
+# SPDX-License-Identifier: MIT
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+    name: {{ .Chart.Name }}
```

Please use an indentation of 2 blanks in _**all**_ `.yaml` files here to be in line with the other Helm charts in our project.
See coding conventions for yaml: https://mercedes-benz.github.io/sechub/latest/sechub-techdoc.html#yaml 
```suggestion
  name: {{ .Chart.Name }}
```

---

I formatted the whole file with only two blanks per intentation.